app/services/queries/reviews_turnaround_per_user_project_query.rb

```diff
@@ -0,0 +1,32 @@
+module Queries
+  class ReviewsTurnaroundPerUserProjectQuery < BaseService
+    attr_reader :time_interval
+
+    def initialize(time_interval:)
+      @time_interval = time_interval
+    end
+
+    def call
+      run
+    end
+
+    private
+
+    def query
+      Events::Review.joins(:review_request, pull_request: :project).includes(:review_request)
+                    .where(opened_at: @time_interval.starting_at..@time_interval.ending_at)
+    end
+
+    def run
+      query.collect do |review|
+        { entity: find_user_project(review.owner, review.pull_request.project),
+          turnaround: [review.review_request.created_at.to_i,
+                       review.opened_at.to_i].reduce(:-) }
```

Comment:
alternative
```suggestion
        {
          entity: find_user_project(review.owner, review.pull_request.project),
          turnaround: review.review_request.created_at.to_i - review.opened_at.to_i
        }
```

Reply:
I did it in that way, but got a line length error from Rubocop...

Reply:
I got an approach: `review.review_request.created_at.to_i.-(review.opened_at.to_i)` then with the parenthesis I can put it in two lines.